app/components/typography/DescriptionWithChildren.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+interface DescriptionWithChildrenProps {
+  text: string;
+  lineHeight: string;
+  letterSpacing?: string;
+  children?: React.ReactNode;
+}
+
+const DescriptionWithChildren: React.FC<DescriptionWithChildrenProps> = ({
+  text,
+  children,
+  lineHeight,
+  letterSpacing,
+}) => {
+  return (
+    <h1
+      className={`font-normal text-sm ${lineHeight} ${letterSpacing} text-secondaryColor`}
+    >
+      {text} {children}
```

Comment:
If children are not always passed, ensure that empty child nodes don’t introduce unnecessary whitespace. Optionally, you can conditionally render them:
```suggestion
      {text} {children && <span>{children}</span>}
```

Reply:
done